bhclub.html

```diff
@@ -44,6 +44,45 @@ <h1>BrickHack</h1>
         </section>
         <section id="about">
             <h1>About</h1>
+            <p>A hackathon is a “hacking marathon” in which teams or individuals have a specified time frame to create something from nothing. These events foster creative problem solving, teamwork, and innovative thinking within the hackathon community.</p>
+            <p>Our flagship event is BrickHack, a yearly 24-hour hackathon event. BrickHack attracts 600+ hackers from around the world each year. For more information about the next Brickhack event, check out <a href="./index.html" target="_blank">our event’s website</a>.</p>
+            <div class="card">
+                <button class="card-header">
+                    Do I need hackathon experience to join BrickHack?
+                    <i class="fa fa-plus" aria-hidden="true"></i>
+                </button>
+                <div class="card-body">
+                    <p>Nope! Having prior experience with hackathons is appreciated but not a requirement.</p>
+                </div>
+            </div>
+            <div class="card">
+                <button class="card-header">
+                    What will I be doing?
+                    <i class="fa fa-plus" aria-hidden="true"></i>
+                </button>
+                <div class="card-body">
+                    <p>What you do with BrickHack will vary depending upon the team that you join.</p>
```

Comment:
I did not have a preset list of FAQs, as I stated in the initial PR comment all of the FAQ content likely needs to be redone and expanded upon. 

I feel that this might introduce the idea that we have different teams within the club, because that is not a given (not all clubs run like this) and it not brought up before this section. But we could leave it for a blub in the teams section or move it to an earlier piece of content.

Reply:
> I did not have a preset list of FAQs, as I stated in the initial PR comment all of the FAQ content likely needs to be redone and expanded upon.

I'm sorry, I didn't see that.

> I feel that this might introduce the idea that we have different teams within the club, because that is not a given (not all clubs run like this) and it not brought up before this section. But we could leave it for a blurb in the teams section or move it to an earlier piece of content.

That makes sense. I agree that there should be something here mentioning teams, but it feels weird to only have this. I don't know, it might just be me parsing it weirdly.

Reply:
Per design, teams are going to be mentioned in the landing section